cmd/edit/cluster/cmd.go

```diff
@@ -34,12 +34,17 @@ import (
 	"github.com/openshift/rosa/pkg/rosa"
 )
 
-const doubleQuotesToRemove = "\"\""
+const (
+	doubleQuotesToRemove = "\"\""
```

Comment:
I think backticks would be more readable:
```suggestion
	doubleQuotesToRemove = `""`
```